modules/ROOT/pages/data-modeling.adoc

```diff
@@ -73,19 +73,12 @@ You should change any top-level user properties that have a key with a leading u
 * Renaming them to remove the underscore, or,
 * Wrapping them inside another object with a key that doesn't have a leading underscore.
 
-
-
-
-
-
```

Comment:
Deleting multiple whitespace lines is fine, but just to note, sometimes people find them useful before a `==` to help break up the structure.
I think *2* newlines is sufficient in that case though.

(Skim-reading an asciidoc is hard enough as it is.)

Anyway, just a thought to put that into context.